UPGRADE-FROM-1.12-WITH-PRICE-HISTORY-PLUGIN-TO-1.13-LEGACY.md

```diff
@@ -0,0 +1,134 @@
+⚙️ Upgrade from Sylius 1.12 with PriceHistoryPlugin to Sylius 1.13
+==================================================================
+
+We encourage you to use the upgrade instructions based on Rector as it is more convenient and faster to accomplish.
+The rector installation guide is available [here](UPGRADE-FROM-1.12-WITH-PRICE-HISTORY-PLUGIN-TO-1.13.md).
+
+Legacy upgrade
+--------------
+
+1. Remove the PriceHistoryPlugin from composer.json by running:
+
+    ```bash
+    composer remove sylius/price-history-plugin --no-scripts
+    ```
+
+1. Make sure to remove the following config from your `config/packages/sylius_price_history_plugin.yaml` file:
+
+    ```diff
+    -  - { resource: "@SyliusPriceHistoryPlugin/config/config.yaml" }
+    ```
+   And the route from your `config/routes/sylius_price_history_plugin.yaml`:
+    ```diff
```

Comment:
Use blank lines before and after code blocks:
```suggestion
    ```

   And the route from your `config/routes/sylius_price_history_plugin.yaml`:

    ```diff
```
here and in other places